x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/migration_status_panels/migration_result_panel.tsx

```diff
@@ -33,88 +37,120 @@ import { SecuritySolutionLinkButton } from '../../../../common/components/links'
 import type { RuleMigrationStats } from '../../types';
 import { RuleTranslationResult } from '../../../../../common/siem_migrations/constants';
 import * as i18n from './translations';
+import { RuleMigrationsUploadMissingPanel } from './upload_missing_panel';
+
+const headerStyle = css`
+  &:hover {
+    cursor: pointer;
+    text-decoration: underline;
+  }
+`;
```

Comment:
Could consider @emotion/react here given this is a new import.

Reply:
Yup, I still have to change this default in my brain settings 🧠. changed here: https://github.com/elastic/kibana/pull/205734/commits/90d829a67e1bedfafc4c30d3d5fe230b4f295af5